VogtPayroll2/EmployeeSalary1.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    class EmployeeSalary : Employee
+    {
+        private decimal _salary;
+
+        public EmployeeSalary(string aName, int aHoursworked, decimal aPayRate)
```

Comment:
Employee does not take in a payrate! They take in a salary.

Change this to:
```c#
public EmployeeSalary(string name, int hoursWorked, decimal salary) : base(name, hoursWorked) {
  _salary = salary;
}

Reply:
base() calls the parent's constructor, btw. So you dont have to handle setting those first 2 parameters (`name` and `hoursWorked`) -- the parent constructor will.